elo_boat.py

```diff
@@ -444,110 +442,104 @@ async def delete_account(ctx):
 async def balance(ctx, *players):
     ELO = []
 
-    # If all players are in the Elobot Voice channel
+    # TODO: If all players are in the Elobot Voice channel
     if len(players) == 0:
         print(client.get_channel(elo_bot_voicechat_id).members)
         return
 
-    if len(players) % 2 == 0:
-        if len(list(players)) != len(set(players)):
-            await ctx.send("Duplicate found")
-            return
-        for player in players:
-            # player is alias
-            if "#" in player:
-                # player is wc3name
-                player = player.lower()
-                query = f"SELECT elo,elo_convergence,suspended FROM `player` WHERE wc3_name = '{player}'"
-                cursor = my_db.cursor()
-                cursor.execute(query)
-                result = cursor.fetchone()
-                if result is not None:
-                    if result[2] != 0:
-                        await ctx.channel.send("Cannot balance this game : account " + player + " has been suspended")
-                        return
-                    ELO.append([player, trueenv.Rating(result[0], result[1])])
-                else:
-                    # using default ELO
-                    await ctx.send(f"**{player}** was not found, using fresh new player elo for that player")
-                    ELO.append([player, trueenv.Rating(start_elo, start_elo_convergence)])
-            elif "<@!" in player:
-                # mention
-                pass
+    if len(players) % 2 != 0:
+        await ctx.send("Enter a even number of players")
+        return
+
+    if len(list(players)) != len(set(players)):
+        await ctx.send("Duplicate found")
+        return
+
+    for player in players:
+
+        # player is wc3name
+        if "#" in player:
+            query = f"SELECT elo,elo_convergence,suspended FROM `player` WHERE wc3_name = '{player.lower()}'"
+            cursor = my_db.cursor()
+            cursor.execute(query)
+            result = cursor.fetchone()
+            if result is not None:
+                if result[2] != 0:
+                    await ctx.channel.send("Cannot balance this game. Account: **" + player + "** has been suspended")
+                    return
+                ELO.append([player, trueenv.Rating(result[0], result[1])])
             else:
-                # alias
-                player = player.lower()
-                query = f"SELECT wc3_name,elo,elo_convergence,suspended FROM `player` WHERE `alias` = '{player}'"
-                cursor = my_db.cursor()
-                cursor.execute(query)
-                result = cursor.fetchone()
-                if result is None:
-                    await ctx.send(f"**{player}** was not found, using fresh new player elo for that player")

```

Comment:
Why would you want to delete this handler ? you just can't balance a game with an unregistered player now ?

Reply:
yes, that is the plan